engine/requirements.txt

```diff
@@ -33,7 +33,7 @@ django-log-request-id==1.6.0
 django-polymorphic==3.0.0
 django-rest-polymorphic==0.1.9
 pre-commit==2.15.0
-https://github.com/grafana/django-push-notifications/archive/refs/tags/3.0.0-fix-migration.tar.gz
+https://github.com/grafana/fcm-django/archive/refs/tags/v1.0.12r1.tar.gz
```

Comment:
I again needed to make a fork because `fcm-django` had the same issue as `django-push-notifications` in that it wouldn't properly allow specifying the foreign key relationship for the `user_id` column (would automatically set this to `auth_user`.. which isn't appropriate here). The forked version has this issue patched.